hnn_core/network_models.py

```diff
@@ -185,11 +185,13 @@ def law_2021_model():
     6) Addition of L5_basket -> L5_pyramidal distal connection
     """
 
-    hnn_core_root = op.dirname(hnn_core.__file__)
-    params_fname = op.join(hnn_core_root, 'param', 'default.json')
-    params = read_params(params_fname)
+    if params is not None:
```

Comment:
shouldn't this be handled in `jones_2009_model` ?

Reply:
My vote is to have all template models have the same API. Are there any reasons why this might be undesirable?

Reply:
you'd have to define param files for each of the template models then ...

Reply:
Also, I believe all previously existing param files work with the other template models, albeit with different effects.

Reply:
for now, just pass `params` directly into `jones_2009_model` whatever you get in `law_2021_model` so it's clear that the param extraction is done only for Jones 2009 model and mention in docstring that we don't support additional parameters than the one provided in Jones (2009) model. So get rid of L188-191 because this is handled here:

https://github.com/jonescompneurolab/hnn-core/blob/0406ed1a2b2335b786e83eb1698f27a5c3dcdadc/hnn_core/network_models.py#L42-L46

Then when you have time, we should remove the additional `params` argument to these template models and enhance the `optimize_evoked` function so it works with any `Network` object.